### PR TITLE
[MM-22294] Fixed the use of handleUploadError on create_comment

### PR DESCRIPTION
**Pull request by devinbinnie**
#### Summary
When a user uploads a file of size greater than the configured maximum upload size, but smaller than the server (or proxy) configured maximum request size, an error was being generated. This exposed a bug in `create_comment` where the server rejecting files due to unknown errors was causing a JS error. This was caused by `handleUploadError()` not taking it the correct number of parameters.

This PR fixes the number of parameters such that `handleUploadError()` will now handle errors correctly.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-22294

### CODE REVIEW

**Comment by amyblais** [comment]
@devinbinnie Is this a regression in v5.20? What caused it?

File: components/create_comment/create_comment.jsx [hunk 1]
```diff
@@ -661,7 +661,7 @@ class CreateComment extends React.PureComponent {
         }
     }
 
-    handleUploadError = (err, clientId = -1, rootId = -1) => {
+    handleUploadError = (err, clientId = -1, currentChannelId, rootId = -1) => {
```
Comment (jespino) [inline components/create_comment/create_comment.jsx]: maybe setting the default value to `-1` to be consistent with the rest of the other parameters?

Comment (devinbinnie) [reply components/create_comment/create_comment.jsx]: Lint complains since the value is never used in this function.

**Comment by devinbinnie** [comment]
> @devinbinnie Is this a regression in v5.20? What caused it?

It's been a bug for a long time, not a regression.
The community server configuration changed limiting the upload file size to less than what was configured in Mattermost. This is being fixed as well, but it exposed this bug where we weren't handling the error correctly on the RHS.

**Comment by amyblais** [comment]
I'll move this to v5.21 since this is not a regression. Let me know any objections cc @jgilliam17

**Comment by jgilliam17** [comment]
> I'll move this to v5.21 since this is not a regression. Let me know any objections cc @jgilliam17

@amyblais  - No objections from QA. Thanks.

**Pull request merged**